integrationTests/package.json

```diff
@@ -8,9 +8,10 @@
     "depcheck": "depcheck",
     "lint": "eslint . --ext .ts,.js --quiet --fix",
     "pretest": "npm run lint && npm run depcheck",
-    "test:ts": "nyc mocha \"tests/*.test.ts\"",
-    "test:js": "nyc mocha \"tests/*.test.js\"",
+    "test:ts": "nyc mocha \"tests/basic/*.test.ts\"",
```

Comment:
I'd rather see this changed to test:basic:ts and test:basic:js. Then futher down test:cache:ts test:cache:js. Potentially introducing test:basic and test:cache to run all basic and cache tests.

Reply:
test:basic:ts and test:basic:js is a good idea, changed it..

we do not need separate cache scripts, as some of the basic tests include native cache tests..